app/views/provider_interface/application_choices/respond.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= form_tag provider_interface_application_choice_new_reject_path, method: :get do %>
```

Comment:
We can use the structure from https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/2fdef039740a86bc2345be7a26498bb94c4c8a5f/app/views/candidate_interface/start_page/eligibility.html.erb for this.